lib/ilo-sdk/helpers/manager_account_helper.rb

```diff
@@ -26,17 +26,12 @@ def get_account_privileges(username)
       end
     end
 
-    # Set the UEFI shell start up
+    # Set the privileges for a user
     # @param [TrueClass, FalseClass] username
-    # @param [TrueClass, FalseClass] login
-    # @param [TrueClass, FalseClass] remote_console
-    # @param [TrueClass, FalseClass] user_config
-    # @param [TrueClass, FalseClass] virtual_media
-    # @param [TrueClass, FalseClass] virtual_media_power_and_reset
-    # @param [TrueClass, FalseClass] ilo_config
+    # @param [Hash] privileges
```

Comment:
Can we add yarddocs for the options that can be set here? See [this](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#option)